requirements-core.txt

```diff
@@ -17,7 +17,7 @@ pip>=18.0
 python-louvain>=0.13
 pyyaml
 requests
-scikit-learn>=1.1.0,<1.2.0
+scikit-learn>=1.1.0,!=1.2.*  # ignoring 1.2.*: scikit-learn/issues/26241
```

Comment:
Is there any specific reason why meta.yaml includes `<1.4` version scikit-learn constraint while core doesn't?

Reply:
Yes. 1.4 deprecates some things that are still used but unfixed by this PR.